image_view/src/stereo_view_node.cpp

```diff
@@ -97,8 +97,9 @@ StereoViewNode::StereoViewNode(const rclcpp::NodeOptions & options)
   std::string format_string = this->get_parameter("filename_format").as_string();
   filename_format_.parse(format_string);
 
-  this->declare_parameter<std::string>("transport", std::string("raw"));
-  std::string transport = this->get_parameter("transport").as_string();
+  // TransportHints does not actually declare the parameter
+  this->declare_parameter<std::string>("image_transport", std::string("raw"));
```

Comment:
NOTE: if we back port this - we probably need to revert the change in name of this parameter